Fix valid start & valid end column formats

alisonsnwong opened the pull request:
https://github.com/UCDavisLibrary/ava/issues/845

Fixed the valid start & valid end column formats bug.

MicheleTobias commented:
This is an excellent start and you've almost solved the issue. Here are some refinements I'd like you to implement:
1. The date format should match the [definitions](https://ucdavislibrary.github.io/ava/data.html) we put in the documentation.
2. The area column needs to be removed. We use that to order the stacking of the polygons (small polygons on top) so we can see all of them, but we don't need to give that to the users. They should calculate their own area in the projection of their choice.

alisonsnwong commented:
I updated the file so that it is in date format and removed the area column. The `removed` column is still listed as a `string` type in QGIS because I believe they are all NULL values but if there is a date added in one of the attributes in the future, it would become the `date` type like listed in R.
